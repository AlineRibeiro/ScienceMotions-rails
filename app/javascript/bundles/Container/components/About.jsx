import React from "react";
import TubeOne from "images/tube-1.png";
import TubeTwo from "images/tube-2.png";
import TubeBright from "images/tube-bright.gif";
import FacesFast from "images/faces-fast.gif";
import { ParallaxProvider, Parallax } from "react-skrollr";

const About = ({}) => (
  <ParallaxProvider>
    <Parallax
      data={{
        "data-0":
          "background-image: linear-gradient(90deg, hsl(172, 100%, 50%), hsl(318, 68%, 68%)",
        "data-450":
          "background-image: linear-gradient(450deg, hsl(172, 100%, 50%), hsl(318, 68%, 68%)"
      }}
    >
      <section className="container center-text">
        <div className="row">
          <div className="col-xs-6 col-md-3 portfolio-item">
            <img className="img-responsive" src={TubeOne} alt="" />
          </div>
          <div className="hidden-xs hidden-sm col-md-3 portfolio-item">
            <img className="img-responsive" src={TubeTwo} alt="" />
          </div>
          <div className="col-xs-6 col-md-3 portfolio-item">
            <img className="img-responsive" src={FacesFast} alt="" />
          </div>
          <div className="hidden-xs hidden-sm col-md-3 portfolio-item">
            <img className="img-responsive" src={TubeBright} alt="" />
          </div>
        </div>
        <div className="row">
          <div className="col-lg-12">
            <div className="header">
              <h1>ScienceMotions</h1>
              <p>Biology, chemistry and physics lessons from the future.</p>
              <p>
                <a href="#">see more</a>
              </p>
            </div>
          </div>
        </div>
        <div className="row">
          <div className="col-xs-6 col-md-3 portfolio-item">
            <img className="img-responsive" src={FacesFast} alt="" />
          </div>
          <div className="hidden-xs hidden-sm col-md-3 portfolio-item">
            <img className="img-responsive" src={TubeTwo} alt="" />
          </div>
          <div className="hidden-xs hidden-sm col-md-3 portfolio-item">
            <img className="img-responsive" src={TubeOne} alt="" />
          </div>
          <div className="col-xs-6 col-md-3 portfolio-item">
            <img className="img-responsive" src={TubeBright} alt="" />
          </div>
        </div>
      </section>
    </Parallax>
  </ParallaxProvider>
);

export default About;
