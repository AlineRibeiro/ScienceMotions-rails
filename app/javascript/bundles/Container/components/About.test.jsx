import React from 'react';
import renderer from 'react-test-renderer';
import About from './About';

describe('About', () => {
  window.scrollTo = jest.fn(() => ({})); 
  it('renders correctly when there are no items', () => {
    const rendered = renderer.create(<About />).toJSON();
    expect(rendered).toMatchSnapshot();
  });
})
