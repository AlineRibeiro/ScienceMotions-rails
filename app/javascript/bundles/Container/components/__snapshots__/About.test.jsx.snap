// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About renders correctly when there are no items 1`] = `
<div
  id="skrollr-body"
>
  <div
    data-0="background-image: linear-gradient(90deg, hsl(172, 100%, 50%), hsl(318, 68%, 68%)"
    data-450="background-image: linear-gradient(450deg, hsl(172, 100%, 50%), hsl(318, 68%, 68%)"
  >
    <section
      className="container center-text"
    >
      <div
        className="row"
      >
        <div
          className="col-xs-6 col-md-3 portfolio-item"
        >
          <img
            alt=""
            className="img-responsive"
            src="test-file-stub"
          />
        </div>
        <div
          className="hidden-xs hidden-sm col-md-3 portfolio-item"
        >
          <img
            alt=""
            className="img-responsive"
            src="test-file-stub"
          />
        </div>
        <div
          className="col-xs-6 col-md-3 portfolio-item"
        >
          <img
            alt=""
            className="img-responsive"
            src="test-file-stub"
          />
        </div>
        <div
          className="hidden-xs hidden-sm col-md-3 portfolio-item"
        >
          <img
            alt=""
            className="img-responsive"
            src="test-file-stub"
          />
        </div>
      </div>
      <div
        className="row"
      >
        <div
          className="col-lg-12"
        >
          <div
            className="header"
          >
            <h1>
              ScienceMotions
            </h1>
            <p>
              Biology, chemistry and physics lessons from the future.
            </p>
            <p>
              <a
                href="#"
              >
                see more
              </a>
            </p>
          </div>
        </div>
      </div>
      <div
        className="row"
      >
        <div
          className="col-xs-6 col-md-3 portfolio-item"
        >
          <img
            alt=""
            className="img-responsive"
            src="test-file-stub"
          />
        </div>
        <div
          className="hidden-xs hidden-sm col-md-3 portfolio-item"
        >
          <img
            alt=""
            className="img-responsive"
            src="test-file-stub"
          />
        </div>
        <div
          className="hidden-xs hidden-sm col-md-3 portfolio-item"
        >
          <img
            alt=""
            className="img-responsive"
            src="test-file-stub"
          />
        </div>
        <div
          className="col-xs-6 col-md-3 portfolio-item"
        >
          <img
            alt=""
            className="img-responsive"
            src="test-file-stub"
          />
        </div>
      </div>
    </section>
  </div>
</div>
`;
